packages/foreman/nodejs-sass-loader/nodejs-sass-loader.spec

```diff
@@ -4,32 +4,15 @@
 %global npm_name sass-loader
 
 Name: %{?scl_prefix}nodejs-sass-loader
-Version: 10.4.1
+Version: 13.3.2
 Release: 1%{?dist}
 Summary: Sass loader for webpack
 License: MIT
 Group: Development/Libraries
 URL: https://github.com/webpack-contrib/sass-loader
-Source0: https://registry.npmjs.org/@types/json-schema/-/json-schema-7.0.15.tgz
-Source1: https://registry.npmjs.org/ajv/-/ajv-6.12.6.tgz
-Source2: https://registry.npmjs.org/ajv-keywords/-/ajv-keywords-3.5.2.tgz
-Source3: https://registry.npmjs.org/big.js/-/big.js-5.2.2.tgz
-Source4: https://registry.npmjs.org/emojis-list/-/emojis-list-3.0.0.tgz
-Source5: https://registry.npmjs.org/fast-deep-equal/-/fast-deep-equal-3.1.3.tgz
-Source6: https://registry.npmjs.org/fast-json-stable-stringify/-/fast-json-stable-stringify-2.1.0.tgz
-Source7: https://registry.npmjs.org/json-schema-traverse/-/json-schema-traverse-0.4.1.tgz
-Source8: https://registry.npmjs.org/json5/-/json5-2.2.3.tgz
-Source9: https://registry.npmjs.org/klona/-/klona-2.0.6.tgz
-Source10: https://registry.npmjs.org/loader-utils/-/loader-utils-2.0.4.tgz
-Source11: https://registry.npmjs.org/lru-cache/-/lru-cache-6.0.0.tgz
-Source12: https://registry.npmjs.org/neo-async/-/neo-async-2.6.2.tgz
-Source13: https://registry.npmjs.org/punycode/-/punycode-2.3.1.tgz
-Source14: https://registry.npmjs.org/sass-loader/-/sass-loader-10.4.1.tgz
-Source15: https://registry.npmjs.org/schema-utils/-/schema-utils-3.3.0.tgz
-Source16: https://registry.npmjs.org/semver/-/semver-7.5.4.tgz
-Source17: https://registry.npmjs.org/uri-js/-/uri-js-4.4.1.tgz
-Source18: https://registry.npmjs.org/yallist/-/yallist-4.0.0.tgz
-Source19: nodejs-sass-loader-%{version}-registry.npmjs.org.tgz
+Source0: https://registry.npmjs.org/neo-async/-/neo-async-2.6.2.tgz
```

Comment:
If it's just a single dependency, it's tempting to package it as non-bundled to avoid including the tarball with JSON responses in our repository history.